src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -281,17 +407,60 @@ private void VerifyDirectorySecurity(DirectorySecurity expectedSecurity)
             info.Create(expectedSecurity);
 
             Assert.True(Directory.Exists(path));
-            Assert.Equal(typeof(FileSystemRights), expectedSecurity.AccessRightType);
 
             DirectoryInfo actualInfo = new DirectoryInfo(info.FullName);
 
             DirectorySecurity actualSecurity = actualInfo.GetAccessControl();
 
-            VerifyDirectoryAccessSecurity(expectedSecurity, actualSecurity);
+            VerifyAccessSecurity(expectedSecurity, actualSecurity);
         }
 
-        private void VerifyDirectoryAccessSecurity(DirectorySecurity expectedSecurity, DirectorySecurity actualSecurity)
+        #endregion
+
+        #region FileInfo Create
+
+        private FileSecurity GetFileSecurity(WellKnownSidType sid, FileSystemRights rights, AccessControlType controlType)
         {
+            FileSecurity security = new FileSecurity();
+
+            SecurityIdentifier identity = new SecurityIdentifier(sid, null);
+            FileSystemAccessRule accessRule = new FileSystemAccessRule(identity, rights, controlType);
+            security.AddAccessRule(accessRule);
+
+            return security;
+        }
+
+        private void VerifyFileSecurity(FileSecurity expectedSecurity)
```

Comment:
Would this be better to be folded into the other method as default parameters?

Reply:
I'm calling this signature in more than one place